.github/workflows/code_style.yaml

```diff
@@ -1,5 +1,5 @@
 name: Code style
-on:
+"on":
```

Comment:
Not how it is done in other YAML files.

Reply:
Yeah, that's for the next PR: https://github.com/timescale/timescaledb/pull/5237
Apparently `on` is a truthy value, and somehow this applies to keys as well, so it's better to quote it. Same as the famous Norway problem :)

Reply:
No strong opinion on doing one way or the other, as long as it is consistent, simple to use, and clear to readers. However, since most of the examples on GitHub does not quote this, people will keep writing the unquoted version, and it will just generate extra work for no specific reason. Is it possible to tweak this?

Reply:
Yes, I can disable this particular warning globally. Not sure how useful it is. We don't often add new workflows though.

Reply:
> We don't often add new workflows though.

Good point. We can also disable it later if it turns out to be a nuisance and cause friction. I have approved the PR, so you can decide what you think is best.